docs/configurations.md

```diff
@@ -0,0 +1,235 @@
+| `scalar.db.storage`                                       | `jdbc` must be specified.                                                                                                                                                    | -                            |
+| `scalar.db.contact_points`                                | JDBC connection URL.                                                                                                                                                         |                              |
+| `scalar.db.username`                                      | Username to access the database.                                                                                                                                             |                              |
+| `scalar.db.password`                                      | Password to access the database.                                                                                                                                             |                              |
+| `scalar.db.jdbc.connection_pool.min_idle`                 | Minimum number of idle connections in the connection pool.                                                                                                                   | `20`                         |
+| `scalar.db.jdbc.connection_pool.max_idle`                 | Maximum number of connections that can remain idle in the connection pool.                                                                                                   | `50`                         |
+| `scalar.db.jdbc.connection_pool.max_total`                | Maximum total number of idle and borrowed connections that can be active at the same time for the connection pool. Use a negative value for no limit.                        | `100`                        |
+| `scalar.db.jdbc.prepared_statements_pool.enabled`         | Setting this property to `true` enables prepared-statement pooling.                                                                                                          | `false`                      |
+| `scalar.db.jdbc.prepared_statements_pool.max_open`        | Maximum number of open statements that can be allocated from the statement pool at the same time. Use a negative value for no limit.                                                  | `-1`                         |
+| `scalar.db.jdbc.isolation_level`                          | Isolation level for JDBC. `READ_UNCOMMITTED`, `READ_COMMITTED`, `REPEATABLE_READ`, or `SERIALIZABLE` can be specified.                                                       | Underlying-database specific |
+| `scalar.db.jdbc.metadata.schema`                          | Schema name for the namespace and table metadata used for ScalarDB.                                                                                                          | `scalardb`                   |
```

Comment:
Since this PR is for `3.8` and earlier, I think this configuration should be as follows, right?

```suggestion
| `scalar.db.jdbc.table_metadata.schema`                          | Schema name for the table metadata used for ScalarDB.                                                                                                          | `scalardb`                   |
```